cocotb/runner.py

```diff
@@ -656,18 +656,19 @@ def _build_command(self) -> List[Command]:
             + [str(source_file) for source_file in self.vhdl_sources]
         ]
 
+        if self.hdl_toplevel is not None:
+            cmds += [
+                ["ghdl", "-m"]
+                + [f"--work={self.hdl_library}"]
```

Comment:
I know you only copied this code, but while you're at it, let's clean this up:

```suggestion
                + ["--work", self.hdl_library"]
```

Reply:
I don't think that's correct. It needs to be [`--work=<LIB_NAME>`](https://ghdl.github.io/ghdl/using/InvokingGHDL.html#cmdoption-ghdl-work).

Reply:
Are you sure? Most command-line parsers accept the version without `=` as well, with the added benefit of getting automatic shell escaping for `self.hdl_library` as value (imagine it containing a `=` itself, which is unlikely here, of course).

Reply:
It requires "--work=" as the prefix:
https://github.com/ghdl/ghdl/blob/a0321bd7ae81196b1b3a260224b0ee5102c32d61/src/options.adb#L187

Reply:
Ugh. Looks like no argument parsing library was available :-) Ignore my comment then, sorry for the noise.